bus/bus.go

```diff
@@ -585,6 +586,22 @@ func (b *bus) setRedundancySettings(rs RedundancySettings) error {
 	}
 }
 
+func (b *bus) contractsAncestorsHandlerGet(jc jape.Context) {
+	var fcid types.FileContractID
+	if jc.DecodeParam("id", &fcid) != nil {
+		return
+	}
+	var minStartHeight uint64
+	if jc.DecodeForm("startHeight", &minStartHeight) != nil {
```

Comment:
prefer to pass arguments via body instead of form values. Form values should be used for optional args (and even then, avoided if possible)